.travis.yml

```diff
@@ -4,14 +4,10 @@ node_js:
 dist: trusty
 addons:
   chrome: stable
+  firefox: latest
 before_install:
   - if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1920x1080x16"
-  # give xvfb some time to start
-  - sleep 3
 before_script:
-  # Same as what we specified for Xvfb.
-  - "export DISPLAY=:99.0"
```

Comment:
Is this needed for screenshots?

Reply:
It was needed for tests run on the virtual display without popping up the browser.
But now headless mode is used and it's not needed.